source/Depreciation.cpp

```diff
@@ -259,18 +262,19 @@ int64_t Depreciation::Value(const Ship *ship, int day, int count) const
 
 
 // Get the value of an outfit.
-int64_t Depreciation::Value(const Outfit *outfit, int day, int count) const
+int64_t Depreciation::Value(const Outfit *outfit, int day, const PlayerInfo *player, int count) const
 {
+	int64_t cost = (player ? player->GetPlanet() ? player->GetPlanet()->GetLocalRelativePrice(outfit, player->Conditions()) : 1 : 1) * outfit->Cost();
```

Comment:
I'm not sure what you mean, I'll push the change you proposed in the meantime as its clearer

Even in that case the conditions and planet of the player would be required (which means actually not the entire player but idk if I should change that one pointer into 2 pointers)
What do you mean, take a pricing list? This method is called at a lot of different places, should they have a pricing list added to their cache? I think it would be a lot of trouble for not much considering the game does not have to do anything whilst in the outfitter so code clarity would be more important than complete optimization?
Also the method that would have to be changed would be the one in Planet but it would mean multiple classes would interact directly with the CustomSales instead of just Planet like it is the case currently.

Reply:
Done!